internal/store/postgres/receiver.go

```diff
@@ -102,7 +104,12 @@ func (r ReceiverRepository) List(ctx context.Context, flt receiver.Filter) ([]re
 		return nil, err
 	}
 
-	rows, err := r.client.QueryxContext(ctx, query, args...)
+	attrs := []attribute.KeyValue{
+		attribute.String("db.method", "Select *"),
+		attribute.String("db.sql.table", "receivers"),
```

Comment:
The attribute table name is static and could be initiated when repository is created